services/web-server/test/session_store_test.js

```diff
@@ -119,13 +118,10 @@ helper.secrets.mockSuite(testing.suiteName(), [], function(mock, skipping) {
         store.touch('foo', { cookie: { maxAge: 300 } }, done);
       });
     });
-    test('it should error when touching the session', async function () {
```

Comment:
Was this intended to test that touching a nonexistent session fails?  Is that an important characteristic?

Reply:
No, I think that was just testing the error message.

Reply:
I retract my comment. I need to fix this.